Library/Homebrew/exceptions.rb

```diff
@@ -235,8 +235,10 @@ def dump
           puts "#{formula} was moved to homebrew-boneyard because it has unfixable issues."
           puts "Please do not file any issues about this. Sorry!"
         else
-          puts "If reporting this issue please do so at (not Homebrew/homebrew):"
-          puts "  https://github.com/#{formula.tap}/issues"
+          if issues_url = formula.tap.issues_url
+            puts "If reporting this issue please do so at (not Homebrew/homebrew):"
+            puts "  #{issues_url}"
```

Comment:
Won't this just be blank for taps using custom remotes? Should we special-case that to have it say something else, like "we don't know where issues for this go" or just replace the blank with a "???" or something?